src/ServiceControl/Persistence/PersistenceHostBuilderExtensions.cs

```diff
@@ -20,7 +20,7 @@ public static IHostBuilder SetupPersistence(this IHostBuilder hostBuilder, Setti
                 serviceCollection.AddSingleton<IDocumentStore>(documentStore);
                 serviceCollection.AddHostedService<EmbeddedRavenDbHostedService>();
                 serviceCollection.AddCustomCheck<CheckRavenDBIndexErrors>();
-                //TODO:serviceCollection.AddCustomCheck<CheckRavenDBIndexLag>();
+                //TODO!:serviceCollection.AddCustomCheck<CheckRavenDBIndexLag>();
```

Comment:
Think this was a mistake, code can be readded an compiles so I propose to do that